### ISSUE TITLE
[accessibility] Convey the disabled state to screen reader users

### ISSUE THREAD

**Issue by Malvoz** [davidchouse/Leaflet.NavBar#6]
When the controls are disabled (`leaflet-control-navbar-back-disabled`, `leaflet-control-navbar-fwd-disabled`) they should be conveyed as such to screen reader users.

This can be achieved using [`aria-disabled="true"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled).

Alternatively, the controls could be taken out of the focus order (remove `href="#"`), and set to [`aria-hidden="true"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden).

**Comment by Malvoz** [comment]
Other resources that you may find helpful:

- [Leaflet guide to accessibility: Test your maps](https://leafletjs.com/examples/accessibility/#test-your-maps)
- [Leaflet Plugin Authoring Guide: Content Accessibility](https://github.com/Leaflet/Leaflet/blob/main/PLUGIN-GUIDE.md#content-accessibility)
